confidant/scripts/migrate.py

```diff
@@ -0,0 +1,19 @@
+import sys
+import logging
+from flask.ext.script import Command
+
+from confidant.app import app
+from confidant.models.blind_credential import BlindCredential
+
+
+app.logger.addHandler(logging.StreamHandler(sys.stdout))
+app.logger.setLevel(logging.INFO)
+
+
+class MigrateSetAttribute(Command):
+
+    def run(self):
+        app.logger.info('Migrating UnicodeSetAttribute in BlindCredential')
```

Comment:
Do you need to enforce that the service is in maintenance mode before doing this?

Reply:
Nah. I think it's OK if we tell folks that it's best if they have it in maintenance mode, but I don't think we need to enforce it.